### ISSUE TITLE
Finer grained attack hierarchy?

### ISSUE THREAD

**Issue by defeo** [issikebrokenyet/issikebrokenyet.github.io#5]
It could be useful to have a finer hierarchy than just poly < subexp < exp. For example, both Delfs-Galbraith and Biasse-Jao-Sankar break the isogeny path problem in quantum exponential time, but the latter is preferred (and so far not showing up in the row, because of the ordering in the .yml file).

**Comment by GiacomoPope** [comment]
One option would be to use approximate L notation for all attack strengths for easy ordering. Dunno how realistic this would be for total coverage though, but would help order the various exp / subexp complexities.

**Comment by defeo** [comment]
Yup, should be doable to keep track of exponents in poly, L, exp complexities. Should suffice to capture most clashes. It's a lower priority, though.

**Comment by defeo** [comment]
I added support for writing complexities in the L(a,c) notation. Not closing this issue because one could still do more (e.g., it's not clear why vOW is better than MITM, with this notation), but I don't anticipate doing so.

**Comment by GiacomoPope** [comment]
This is cool! Agree about the finer finer grained details of things like space / time complexity. If I have a smart idea I'll come back here but nothing comes to mind without creating too much complicated notation (unless memory was somehow backend only to allow selection for best attack)
